references/cuegen/util.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2023 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cuegen
+
+import (
+	"fmt"
+	gotypes "go/types"
+	"strconv"
+
+	cueast "cuelang.org/go/cue/ast"
+	cuetoken "cuelang.org/go/cue/token"
+)
+
+var _ = ident("", true) // nolint. will be used in the future. TODO(iyear): remove
```

Comment:
Can be removed now?

Reply:
Related code will be committed in another PR.

Reply:
Updated